node-deb

```diff
@@ -803,6 +803,11 @@ if ! [ -f "$deb_dir/usr/share/$package_name/app/package.json" ]; then
   cp -pf './package.json' "$deb_dir/usr/share/$package_name/app/"
 fi
 
+if [ -f './package-lock.json' ] && ! [ -f "$deb_dir/usr/share/$package_name/app/package-lock.json" ]; then
```

Comment:
I think this deserves to be in a separate PR because the other two items are definitely good candidates for merge, but this one needs a bit of discussion and research. I'm not sure how this will impact the `--install-strategy` option, and it's something I'll have to look into.